geps/gep-713/metadata.yaml

```diff
@@ -2,13 +2,14 @@ apiVersion: internal.gateway.networking.k8s.io/v1alpha1
 kind: GEPDetails
 number: 713
 name: Metaresources and Policy Attachment
-status: Memorandum
+status: Standard
```

Comment:
No problems here, but should add an `obsoletes` section to `relationships` for GEP-2648 and GEP-2649.